app/commands.py

```diff
@@ -373,8 +363,8 @@ async def top(ctx: Context) -> Optional[str]:
     if (args_len := len(ctx.args)) not in (1, 2):
         return "Invalid syntax: !top <mode> (player)"
 
-    if ctx.args[0] not in GAMEMODE_STRINGS:
-        return f'Valid gamemodes: {", ".join(GAMEMODE_STRINGS)}.'
+    if ctx.args[0] not in GAMEMODE_REPR_LIST:
+        return f'Valid gamemodes: {", ".join(GAMEMODE_REPR_LIST)}.'
```

Comment:
TODO: missing a check here for impossible modes - sorta a bug as it will just say "No scores."